src/Microsoft.ApplicationInsights/Managed/Shared/Extensibility/Implementation/Tracing/BaseDefaultHeartbeatPropertyProvider.cs

```diff
@@ -0,0 +1,176 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Tracing
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Reflection;
+    using System.Text;
+    using System.Threading.Tasks;
+#if NETSTANDARD1_3
+    using System.Runtime.InteropServices;
+#endif
+
+    internal class BaseDefaultHeartbeatPropertyProvider : IHeartbeatDefaultPayloadProvider
+    {
+        internal readonly List<string> DefaultFields = new List<string>()
+        {
+             "runtimeFramework",
+            "baseSdkTargetFramework",
+            "osType",
```

Comment:
In the PR in web sdk repo, there is a field called 'osType'. Did you meant it to be same where Azure one overrides this when available?

Reply:
A interesting! I had missed this. There is two, one here and one there. The defined behavior then is that this one will be present, and the one from the Web SDK repo will be ignored silently. 

The name 'osType' is something I can control here, but I cannot control it in the Web SDK. Therefore I would have to change this name to something else for the Web SDK to provide (essentially the same) information. 

What I will do is I will revisit this issue when I get to the Web SDK PR, and either choose to rename the osType name there with a specific override, or come to some other solution. Since this is essentially redundant information though, I bet we just leave it as is and ignore the Azure Instance Metadata value.